code/modules/projectiles/guns/ballistic.dm

```diff
@@ -474,7 +474,7 @@
 	return TRUE
 
 /obj/item/gun/ballistic/process_fire(atom/target, mob/living/user, message = TRUE, params = null, zone_override = "", bonus_spread = 0)
-	var/could_it_misfire = chambered && chambered.can_misfire
+	var/could_it_misfire = can_misfire || chambered.can_misfire
```

Comment:
don't we still need to check that the chambered round exists?

Reply:
no, it always should be present at this point, we don't check for it below either